util/vkgcElfReader.h

```diff
@@ -392,7 +392,7 @@ struct MsgPackIterator {
 };
 
 // =====================================================================================================================
-// Represents a reader for loading data from an [Executable and Linkable Format (ELF)](http://tinyurl.com/2toj8) buffer.
+// Represents a reader for loading data from an [Executable and Linkable Format (ELF)] buffer.
```

Comment:
This broke the link

Reply:
@JaxLinAMD If you want to properly remove the markdown reference, you should also remove the `[` `]` surrounding `Executable and Linkable Format (ELF)`

Reply:
will do it